endpoints.json

```diff
@@ -23,5 +23,74 @@
         }
       ]
     }
+  },
+  "GET /api/users":{
+    "description":"serves an array of users",
+    "queries":[],
+    "exampleResponse":{
+      "users":[{
+      "username": "butter_bridge",
+      "name": "jonny",
+      "avatar_url":
+        "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg"
+    }]}
+  },
+  "GET /api/articles/:article_id":{
+    "description":"serves an article object",
+    "queries": ["article_id"],
```

Comment:
article_id should not be in queries here. A query would be at the end of a url i.e. /api/articles?sort_by=author, in this case the query is sort_by. This endpoint should not take any queries but does have a parametric endpoint, which is showed on line 38 by adding the : before article_id.